che-incubator/che-code
# chore: Update extension icon

### What does this PR do?

Applies a bit better icon for built-in extensions.

Before

![Screenshot from 2024-09-09 15-58-21](https://github.com/user-attachments/assets/3d68d103-cb5c-45e0-8e1c-25b504ea7deb)

After

![Screenshot from 2024-09-09 15-58-35](https://github.com/user-attachments/assets/f84defd6-7d40-4e7d-9633-fe4c2c4dd519)


### What issues does this PR fix?



### How to test this PR?

- Create a workspace using the editor built by this pull request  check
- Go to Extensions view
- Using the filter, show built-in extensions 

### Does this PR contain changes that override default upstream Code-OSS behavior?
